packages/core/aqua-compiler/src/utils.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ArrowWithoutCallbacks, NonArrowType, ProductType } from '@fluencelabs/interfaces';
+import { readFile } from 'fs/promises';
+import path from 'path';
```

Comment:
just for consistency sake I suggest not using qualified imports like `import path from 'path'`

and always use unqulified imports like `import { readFile } from 'fs/promises'`

it's a bit shorter and these things rarely clash with what you have in code so my opinion - no need to qualify imports

not a hard requirement, just a suggestion to keep things consistent if you wish so personally

Reply:
Not agree with this. It totally doesn't matter

Reply:
I think most people would agree that consistency is a good thing in general

Reply:
really? didnt know about that. Maybe should prefer qualified, should check about that. Still, as you said it yourself, this lib is not designed for browser